README.md

```diff
@@ -172,11 +172,18 @@ There are 5 different environment tiles, of which some have different styles of
 #### Locations
 As of January 2022, there are a total of 29 different locations in game.
```

Comment:
We should probably update this to be 30, but we can leave that for a different PR.

Reply:
Yeah, I've done that (and bumped the date) in a different PR, I meant to also add Porto to the list there but got carried away